pkg/v1/providers/infrastructure-azure/v1.0.0/ytt/overlay.yaml

```diff
@@ -119,6 +119,20 @@ spec:
       frontendIPs:
       - name: #@ "lb-{}-ip-frontend".format(data.values.CLUSTER_NAME)
         privateIP: #@ data.values.AZURE_FRONTEND_PRIVATE_IP
+
+    #@ if data.values.AZURE_ENABLE_CONTROL_PLANE_OUTBOUND_LB:
+    #@overlay/match missing_ok=True
+    controlPlaneOutboundLB:
+      frontendIPsCount: #@ data.values.AZURE_CONTROL_PLANE_OUTBOUND_LB_FRONTEND_IP_COUNT
+    #@ end
+
+    #@ if data.values.AZURE_ENABLE_NODE_OUTBOUND_LB:
+    #@overlay/match missing_ok=True
+    nodeOutboundLB:
+      frontendIPsCount: #@ data.values.AZURE_NODE_OUTBOUND_LB_FRONTEND_IP_COUNT
+      idleTimeoutInMinutes: #@ data.values.AZURE_NODE_OUTBOUND_LB_IDLE_TIMEOUT_IN_MINUTES
```

Comment:
is this needed?

Reply:
Upstream documentation - https://capz.sigs.k8s.io/topics/node-outbound-lb.html

```
apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
kind: AzureCluster
metadata:
  name: my-public-cluster
  namespace: default
spec:
  location: eastus
  networkSpec:
    apiServerLB:
      type: Public
    nodeOutboundLB:
      frontendIPsCount: 3
      idleTimeoutInMinutes: 4
```